code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -367,14 +355,14 @@
 	storage_slots = 6
 	req_access = list(ACCESS_VIROLOGY)
 
-/obj/item/storage/lockbox/vials/New()
+/obj/item/storage/lockbox/vials/Initialize(mapload)
 	..()
 	update_icon()
```

Comment:
I don't think this one is required anymore. Test it just in case though

Reply:
The `update_icon()` here is still needed because the sprite for the box is made up of different parts.